sympy/stats/crv_types.py

```diff
@@ -1440,7 +1440,12 @@ class GumbelDistribution(SingleContinuousDistribution):
 
     def pdf(self, x):
         beta, mu = self.beta, self.mu
-        return (1/beta)*exp(-((x-mu)/beta)+exp(-((x-mu)/beta)))
+        z = (x-mu)/beta
+        return (1/beta)*exp(-(z+exp(-z)))
```

Comment:
Maybe just `exp(-(z + exp(-z)))/beta`. (Consider adding spaces around binary operators `+` and `-`.)

Reply:
I forgot to do that here. Thanks for correcting.